src/datePicker/index.unit.tsx

```diff
@@ -1,22 +1,24 @@
-import React from 'react'
-import moment from 'moment'
-import DatePicker from 'datePicker'
+// import React from 'react'
```

Comment:
If you comment all of the tests, we cannot know how this is supposed to be working, and how we could migrate it to SPA

Reply:
Tests have been added ;)